server/svix-server/src/v1/endpoints/event_type.rs

```diff
@@ -192,6 +213,10 @@ async fn list_event_types(
         query = query.filter(eventtype::Column::Name.gt(iterator));
     }
 
+    if let Some(flags) = flags.allowed_feature_flags() {
+        query = query.filter(eventtype::Column::FeatureFlag.is_in(flags.0.iter().cloned()));
```

Comment:
Do we really need to clone? Why not into_iter? Or rather, can we just steal this?

Reply:
`into_iter` won't work as is. `allowed_feature_flags` returns a borrow into `flags` and `into_iter` would try to consume it. I guess we can make `allowed_feature_flags` take `self` because it's unlikely you'd want to use the original after calling that.